spec/ruby/core/thread/inline_rescue_spec.rb

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../shared/kernel/raise', __FILE__)
+
+describe "inline rescue" do
+  it "properly executes compound assignment operators" do
```

Comment:
This should be something along the lines of "does not overwrite a variable when using an inline rescue". The idea is to clearly state that:

1. The spec should do something that _doesn't_ overwrite a value
2. This should not occur when using an inline rescue